### PR TITLE
Add course review routes

### PR DESCRIPTION
**Pull request by terror**

### CODE REVIEW

**Comment by terror** [comment]
@JeffDotPng Will need to add more fields later on but this is a good first rough implementation, need to also think about writing tests for these axum routes.

File: src/user.rs [hunk 1]
```diff
@@ -6,6 +6,12 @@ pub(crate) struct User {
   mail: String,
 }
 
+impl User {
```
Comment (JeffDotPng) [inline src/user.rs:9]: Should this consume the user object?

Comment (JeffDotPng) [inline src/user.rs:9]: Should id() consume the user object?

**Review by JeffDotPng — APPROVED** [review]
Looks good, added an extra test and made the code for review routes slightly prettier.
Will probably merge master into `course-page` soon so we can try out some integration.

**Pull request merged**